src/Eager.ts

```diff
@@ -292,3 +292,6 @@ MissingTerms.doExport();
 
 import { swapVar } from './SwapVar';
 swapVar(this);
+
+import { QuerySuggestPreview } from './ui/QuerySuggestPreview/QuerySuggestPreview';
```

Comment:
This needs to be moved before the `swapVar` thingy do to the export properly, with the other components.